modules/ROOT/pages/release-notes.adoc

```diff
@@ -27,7 +33,4 @@ xref:6.0-release-notes-overview.adoc[{productname} 6.0]
 
 Release notes for {productname} 6.0
 
-// Dummy cell to ensure all rows are rendered
-a|
-
```

Comment:
This probably should not have been removed, as it just won't render if there is not enough cells and without this it'll break when we do the next release. At the very least we should be keeping it and just commenting it out instead.

Reply:
Added dummy table cell back, this time within a comment block with edited comments re future use.

To wit:

////
Dummy table cell.

Remove the comment markup around this (and these comments) when the number of cells in the table is odd.
a|
////